datahub-web-react/src/app/entity/shared/tabs/Dataset/AccessManagement/AccessManagerDescription.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Typography } from 'antd';
+
+export type Props = {
+    description: any;
+};
+
+export default function AccessManagerDescription({ description }: Props) {
+    const DescriptionContainer = styled.div`
```

Comment:
please place these styled components at the top of the file (similar to other components)

Reply:
done update